src/lib/helpers/handleFileTracking.js

```diff
@@ -22,15 +22,13 @@ const handleFileTracking = ({ functionsPath, publishPath }) => {
     const trackingFile = readFileSync(trackingFilePath, 'utf8')
     const [trackedFunctions, trackedPublish] = trackingFile.split(TRACKING_FILE_SEPARATOR)
     const cleanConfiguredFiles = (trackedFiles, dirPath) => {
-      trackedFiles.forEach((file) => {
-        const filePath = join(dirPath, file.trim('\r'))
-        if (file !== '') {
```

Comment:
I'm not sure but the intention here might be to checked the trimmed version of `file`

Reply:
trackingFile.split would sometimes insert '' into trackedFiles; this is just to skip that

Reply:
in these cases, with file === '', it removes the current dir